openedx-wasm/runner.py

```diff
@@ -0,0 +1,72 @@
+from wasmtime import Config, Engine, Linker, Module, Store, WasiConfig
```

Comment:
Just ignore this file for the sake of this PR–it was mostly copy-pasted from an example gist someone had put up, and I used it for my very first attempts to start the wasm python.